draft-ietf-tls-tls13.md

```diff
@@ -2419,7 +2445,44 @@ the existing behavior for SignedCertificateTimestamps {{?RFC6962}},
 and is more easily extensible in the handshake state machine.
 
 
-## Server Parameters Messages
+### PSK Binder
+
+%%% Authentication Messages
+       struct {
+           opaque binding_data[Hash.length];
+       } PskBinder;
+
+The "psk_binder" extension is used to carry a binding between the PSK
+and the current handshake. as well as between the session where the
```

Comment:
Would it be OK to replace the following sentence 

--
The "psk_binder" extension is used to carry a binding between the PSK and the current handshake. as well as between the session where the PSK was established and the session where it was used.
-- 
with 
-- 
The "psk_binder" extension carries the output of an HMAC-based digest computed over the portion of the handshake transcript up to but not including PskBinder.binding_data as the Handshake Context. As such, the PskBinder.binding_data is computed using the same way as the Finished.verify data value, using the PSK as the base key and. If the handshake includes a HelloRetryRequest, the initial ClientHello and HelloRetryRequest are included in the transcript.  The actual ClientHello, complete with the full "psk_binder" extension is included in all other handshake hash computations.
-- 

I would move the text that relates more to the security properties to the end of the section, such as this text: 
"
This requires that both the KDF used to produce the RMS
 +and the MAC used to compute the psk_binder be collision
 +resistant. These are properties of HKDF and HMAC respectively when
 +used with collision resistant hash functions and producing output of
 +at least 256 bits.  Any future replacement of these functions MUST
 +also provide collision resistance.
"